lib/goth/token.ex

```diff
@@ -282,6 +279,21 @@ defmodule Goth.Token do
     }
   end
 
+  defp request({mod, _} = config, method, url, headers, body, opts) when is_atom(mod) do
+    Goth.HTTPClient.request(config, method, url, headers, body, opts)
+  end
+
+  defp request({func, initial_state}, method, url, headers, body, options) do
+    request_options = [method: method, url: url, headers: headers, body: body]
+
+    opts =
+      initial_state
+      |> Keyword.merge(options)
```

Comment:
Oh, this `options` variable was never used, sorry about that. While at it, do you mind removing it?